Assets/MixedRealityToolkit/Interfaces/InputSystem/IMixedRealityInputEventSystem.cs

```diff
@@ -0,0 +1,410 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    public interface IMixedRealityInputEventSystem
```

Comment:
a new interface for this?

what happened to the existing one?

Reply:
Oh I see, it was just moved.

Reply:
Was there any actual changes to this class or was it just moved?

Reply:
No this is indeed a new interface.

IMixedRealityInputEventSystem is different from IMixedRealityInputSystem, in that IMixedRealityInputSystem implements IMixedRealityInputEventSystem and  IMixedRealityEventSystem.

This is just a logical split of the events, in such a way that IMixedRealityInputEventSystem contains all the event responses listed in IMixedRealityEventSystem. This allows another class, like the MixedRealityGlobalInputEventSystemProfile, to implement only the input event methods.